ecf/include/envir-p1.h

```diff
@@ -26,21 +26,21 @@ if [[ ! " prod para test " =~ " ${envir} " && " ops.prod ops.para " =~ " $(whoam
 
 # Developer configuration
 PTMP=/lfs/h2/emc/ptmp
-PSLOT=${hafs_ver:-hafs.v1.0.0}
-SID=${SID:-13L}
+PSLOT=ecflow_hafs
 export COMROOT=${PTMP}/${USER}/${PSLOT}/para/com
-export COMPATH=${PTMP}/${USER}/${PSLOT}/para/com/hafs
+export COMPATH=${COMROOT}/hafs
+export COMhafs=$(compath.py hafs/v1.0.0)/${RUN}.${PDY}/${cyc}
 export COMOUT_PREP="$(compath.py obsproc/v1.1.0)"
 if [ -n "%PDY:%" ]; then
   export PDY=${PDY:-%PDY:%}
   export CDATE=${PDY}%CYC:%
 fi
-export DATAROOT=/lfs/h2/emc/stmp/${USER}/hafs
+export COMINmsg=$(compath.py hafs/v1.0.0)/inp${RUN}
+export DATAROOT=/lfs/h2/emc/stmp/${USER}/hafs/${PSLOT}
 mkdir -p ${DATAROOT} # ${COMhafs}
 export COMINgfs=/lfs/h2/emc/hafstemp/CANNED_input_for_HAFS/com/gfs/v16.3
 export COMINgdas=/lfs/h2/emc/hafstemp/CANNED_input_for_HAFS/com/gfs/v16.3
 export COMINrtofs=/lfs/h2/emc/hafstemp/CANNED_input_for_HAFS/com/rtofs/v2.3
 export COMINobs=/lfs/h2/emc/hafstemp/CANNED_input_for_HAFS/com/obsproc/v1.1
 export DCOMROOT=/lfs/h2/emc/hafstemp/CANNED_input_for_HAFS/dcom
 export COMINobsproc=/lfs/h1/ops/prod/com/obsproc/v1.1/%RUN:%.${PDY}/%CYC:%/atmos
```

Comment:
The COMINobsproc line is not needed and not used in HAFSv1 workflow. Please delete this line.